src/coreclr/System.Private.CoreLib/src/Internal/Runtime/InteropServices/ComActivator.cs

```diff
@@ -91,10 +91,16 @@ public struct ComActivationContext
         public string AssemblyName;
         public string TypeName;
 
+        [RequiresUnreferencedCode("In-built COM support is not trim compatible")]
```

Comment:
We use consitent name for this. This change has a mix of "built-in" and "inbuilt" (it should be without the dash according to https://www.merriam-webster.com/dictionary/inbuilt).

"built-in" seems to be much more common so I would use that. For example, it is the word to use according to http://oreillymedia.github.io/production-resources/styleguide/.